internal/libs/typconv/address.go

```diff
@@ -0,0 +1,9 @@
+package typconv
+
+import (
+	"encoding/hex"
+)
+
+func AddressBytesToString(addr []byte) string {
```

Comment:
This is Ethereum specific AFAIU. Maybe rename it to indicate this?

Reply:
it's not evm specific, just hex encoding.